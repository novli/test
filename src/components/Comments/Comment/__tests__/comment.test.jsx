import { CommentComponent } from '../';

const setup = () => {
  const props = {
    id: 1,
    name: 'Name',
    body: 'Body',
    email: 'e@mail.ru',
  };

  const wrapper = shallow(<CommentComponent {...props} />);

  return {
    props,
    wrapper,
  };
};

describe('components', () => {
  describe('Comment', () => {
    test('component is rendered', () => {
      const { wrapper } = setup();
      expect(wrapper).toMatchSnapshot();
    });
  });
});
