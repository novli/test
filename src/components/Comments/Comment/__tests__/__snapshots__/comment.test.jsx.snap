// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Comment component is rendered 1`] = `
<tr>
  <td>
    1
  </td>
  <td>
    Name
  </td>
  <td>
    Body
  </td>
  <td>
    e@mail.ru
  </td>
</tr>
`;
